### ISSUE TITLE
🐞 [Bug]: (Mainnet Verification) All the links in the grid3_client package are outdated

### ISSUE THREAD

**Issue by A-Harby** [threefoldtech/tfgrid-sdk-ts#3591]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### which package/s did you face the problem with?

grid_client

### What happened?

All the links in the grid3_client package are outdated, and the steps are hard to follow and very unclear.
 
https://www.npmjs.com/package/grid3_client

### which network/s did you face the problem on?

Dev, Main

### version

2.1.0-rc2

### Relevant screenshots/screen records

![image](https://github.com/user-attachments/assets/5ba1166b-5cea-4a1a-b1bb-8ac2165bf746)

### Relevant log output

```console
.
```

**Comment by ramezsaeed** [comment]
- update docs
- create a new issue to add workflow

**Comment by fatmaebrahim** [comment]
### Work In Progress: 

- [ ] update links in README.md

- [x] add directory to the repository object in package.json

Note: Same issue exists in the README.md of the last version

**Comment by fatmaebrahim** [comment]
### Work Complete: 
after research, the problem was found inside the package.json for the grid_client package
added the missing directory key in the repository object 
created a monorepo using lerna to test the solution

PS. we may need to apply this for other packages as in monorepo we should add the directory key to the inner package.json

**Comment by fatmaebrahim** [comment]
This needs to be verified after the next release

**Comment by khaledyoussef24** [comment]
https://www.npmjs.com/package/grid3_client

links in the grid 3 client package still has the same outdated links

<img width="1680" alt="Image" src="https://github.com/user-attachments/assets/5b6d926d-0544-46c4-8b1a-0d0fd6e8cca8" />
<img width="1680" alt="Image" src="https://github.com/user-attachments/assets/2612deba-d58f-488b-9bd4-04145d44f125" />

**Comment by 0oM4R** [comment]
The correct npm package to test is https://www.npmjs.com/package/@threefold/grid_client
and on the latest version 2.8.0 
The mentioned one is outdated we should delete it

**Comment by khaledyoussef24** [comment]
Verified all the links in the new package are working fine


<img width="1680" alt="Image" src="https://github.com/user-attachments/assets/95443788-921f-4dd1-82bd-6815bb851c5b" />
<img width="1680" alt="Image" src="https://github.com/user-attachments/assets/490c5aad-0ecd-42e1-9077-36cf6d55f0ab" />


Shall I make another issue to check for the deletion of the old link?

**Comment by 0oM4R** [comment]
> Shall I make another issue to check for the deletion of the old link?

yes please and deleting the package or refance the new one on npm
